packages/tangle/messagefactory_test.go

```diff
@@ -22,6 +23,7 @@ import (
 
 const (
 	targetPOW     = 10
+	powTimeout    = 10 * time.Second
```

Comment:
lets use 60s here. with this new change we dont have problems with timestamps getting to old anymore

Reply:
here 10s is fine as it is for the test where we set the difficulty to be only 2